docker/server/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "Applying database migrations"
+python manage.py migrate --noinput
```

Comment:
not sure whats the best way to do this, this is following what was done in WAIS; it automatically created a user and ran migrations then. Leads to problems when volunteers migrate on one branch then switch to another branch with different migrations because the database is persisted in a volume. 

plus point of not doing it all automatically is that backend volunteers learn how to read docs and the commands to run migrations and the other stuff behind `python manage.py` . Thoughts?

Reply:
maybe the automation part isnt a bad option for frontend peeps where they can just fire up a backend to work with, but could show backend teams how to start just the pg container with `docker compose run db` + make a copy of environment variables so their local shell is connected to the container?

Reply:
> plus point of not doing it all automatically is that backend volunteers learn how to read docs and the commands to run migrations and the other stuff behind python manage.py . Thoughts?

well it mostly happens for users that are doing some database stuff of their own, and moving from one change to another...

Reply:
i think for the majority of the folks, they're not modifying a database model (similar to another person). This also brings a better awareness of "database migration merge conflicts"

Reply:
tbh, the only time database model change so much is at the start. When most of it is already done, it's just minor changes here and there